BP/history/provinces/bp_k_tripolitania.txt

```diff
@@ -1,10 +1,28 @@
-﻿# k_tripolitania
+﻿# Notes:
+# Updating Christianity to better match Christian history map:
+# https://en.wikipedia.org/wiki/History_of_Christianity#/media/File:Spread_of_Christianity_to_AD_600_(1).png
```

Comment:
Note that according to the other map I linked yesterday, the area around Leptis Magna was converted in the 4th century. This map has less detail.

Reply:
I've put together a proposal with a3dc21f. Donatism in and around k_africa (where it would later influence the rise of Kharijites/Ibadi groups), Audianism in Libya, and Valentinianism/Nicene in Mauretania.